pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -244,6 +244,7 @@
   call_to_action_share_on_social_media: "Share on social media"
   call_to_action_explore_all_hoc_activities: "Explore all Hour of Code activities from Code.org"
   call_to_action_start_dance_party: "Start Dance Party"
+  call_to_action_start_dance_party_ai_edition: "Start Dance Party: AI Edition"
```

Comment:
Are these a priority to be translated?

Reply:
I think at this point we're ok with things not being translated for asap, especially since this is on the code.org site and not hourofcode.com.